src/main/java/org/java/practice/Arrays.java

```diff
@@ -39,7 +61,16 @@ public int getSumAllNumbersFromArray(int[][] numbers) {
      * @return average number of the numbers
      */
     public int getAverageNumFromArray(int[][] numbers) {
-        return -1;
+        int sum = 0;
+        int countElements = 0;
+
+        for (int i = 0; i < numbers.length; i++) {
+            for (int j = 0; j < numbers[i].length; j++) {
+                sum += numbers[i][j];
+                countElements++;
```

Comment:
Can we add the length of every sub-array in the first loop instead of incrementing it every time in the second loop?

Reply:
Done